src/ui/common/src/components/integrations/addIntegrations.tsx

```diff
@@ -148,20 +162,231 @@ const AddIntegrationListItem: React.FC<AddIntegrationListItemProps> = ({
     </Box>
   );
 
+  if (service !== 'Kubernetes') {
+    return (
+      <Box key={service}>
```

Comment:
Can you abstract this out into it's own component please?

Reply:
Yeah I'm tracking it in another Linear task: https://linear.app/aqueducthq/issue/ENG-2876/refactor-on-demand-k8s-registration-flow-into-a-separate-component

Reply:
Great. Thanks for filing a task for this.